### PR TITLE
Adding kube dns benchmark ci job

### PR DESCRIPTION
**Pull request by krzysied**

### CODE REVIEW

File: config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml [hunk 1]
```diff
@@ -685,3 +685,36 @@ periodics:
...
+      - --repo=k8s.io/perf-tests=master
+      - --root=/go/src
+      - --timeout=140
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-leaked-resources
+      - --cluster=kube-dns-benchmark
+      - --extract=ci/latest
+      - --gcp-nodes=3
+      - --gcp-project-type=scalability-project
```
Comment (wojtek-t) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml]: We don't need scalability project, if the cluster is supposed to have 3 nodes.
Scalability projects have 100-node quota, so we don't want to waste them if not needed.

Comment (krzysied) [reply config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml]: I removed `scalability-project` requirement. I assume boskos will find suitable project. Or is it better to assign specific project?

**Comment by wojtek-t** [comment]
@krzyzacy @BenTheElder - can one of you approve the testgrid part?

**Pull request merged**